example/app.js

```diff
@@ -19,17 +19,10 @@ tabGroup.addTab(Titanium.UI.createTab({
 	window: require('facebook_share').window()
 }));
 
-if(Ti.Platform.osname !== 'android') {
```

Comment:
Don't we still need this guard? I don't think we support the Places API on Android.